pages/sign-in/index.page.tsx

```diff
@@ -0,0 +1,57 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+import classNames from "classnames/bind";
+
+import styles from "./index.module.scss";
+
+const cn = classNames.bind(styles);
+
+const KAKAO_REDIRECT_URI = "http://localhost:3000/sign-in/kakao";
+const KAKAO_AUTH_URL = `https://kauth.kakao.com/oauth/authorize?client_id=${process.env.NEXT_PUBLIC_REST_API_KEY}&redirect_uri=${KAKAO_REDIRECT_URI}&response_type=code`;
+
+export default function SignIn() {
+  const router = useRouter();
+
+  const handleKakaoButtonClick = () => {
+    router.push(KAKAO_AUTH_URL);
+  };
+
+  return (
+    <>
+      <header className={cn("header")}>
+        <Image src="/icons/tickita-logo.svg" alt="logo" width={60} height={22} />
```

Comment:
```suggestion
        <Image src="/icons/tickita-logo.svg" alt="logo" width={60} height={22} />
```
`alt`는 `html lang`에 맞게 한국어로 작성하는게 좋을 것 같아요!

Reply:
오 그렇네요... 디테일 잘 잡아주셔서 감사합니다🥹🥹🥹 바로 수정할게요!!